src/pages/posting/components/PassPosting.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import postCreateNewPost from '@apis/posting';
+import { POSTING_DESCRIPTION } from '@pages/posting/constants';
+import { makeFormData } from '../utils';
+import { StyledPassPosting } from './PassPosting.style';
+
+interface PassPostingProps {
+  channelId: string;
+  customToken: string;
+}
+
+const PassPosting = ({ channelId, customToken }: PassPostingProps) => {
+  const navigate = useNavigate();
+  const [showConfirm, setShowConfirm] = useState(false);
+  const { PASS_POSTING } = POSTING_DESCRIPTION;
+
+  const handleClickPassPost = () => {
+    const formData = makeFormData('', channelId);
+    postCreateNewPost(customToken, formData);
+    navigate('/posts');
+  };
```

Comment:
p5;
```suggestion
    const handleClickPassPost = async () => {
    const formData = makeFormData('', channelId);
    await postCreateNewPost(customToken, formData).then(() => {
      navigate('/posts');
    });
  };
```
이런식으로해야 추후에 반환값에 대한 에러처리가 가능할것 같아요!
아니면 지금처럼 요청이 완료되기 전에 포스팅으로 넘어가는게 사용성은 더 좋을지도 모르겠네요? 어떻게 생각하시나요?

Reply:
```suggestion
  const handleClickPassPost = () => {
    const formData = makeFormData('', channelId);
    postCreateNewPost(customToken, formData)
    .then(() => {
      navigate('/posts');
    })
    .catch((error) => {
      console.error(error); // or navigate('/error-500')
    });
  };
```

Reply:
좋네요! 비동기 처리 추가해놓겠습니다!